pkg/controller/tidbcluster/tidb_cluster_controller.go

```diff
@@ -76,11 +76,11 @@ func NewController(
 	tikvFailoverPeriod time.Duration,
 	tidbFailoverPeriod time.Duration,
 ) *Controller {
-	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartLogging(glog.Infof)
+	eventBroadcaster := record.NewBroadcasterWithCorrelatorOptions(record.CorrelatorOptions{QPS: 10})
```

Comment:
the default `QPS` is too low: `1/ 300`.

Reply:
10 events per object per seconds seems too large, it might generate a lot of load on the API server or unable to do other CRUD requests because of exceeding the total throttling limit. Most burstable tokens (25) are consumed by [unnecessary events](https://github.com/pingcap/tidb-operator/issues/1471). We can increase the burst and QPS together but not too much at the first.

Reply:
Yeah, 10 is too large.

I will decrease it to a proper value after removing [those events](https://github.com/pingcap/tidb-operator/issues/1471).

Reply:
what's the current rate of events?

Reply:
`rate of events`, does it mean the default value of `QPS` and `Burstable`?

Reply:
create an issue about setting the rate to proper values?

Reply:
https://github.com/pingcap/tidb-operator/issues/1494

an issue opened